.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+source=matplotlib
+[report]
+omit =
+    */python?.?/*
```

Comment:
To actually get a cover report locally I had to remove  this line  (*/python?.?/*)